hs_communities/models.py

```diff
@@ -0,0 +1,43 @@
+from django.db import models
+
+
+class Topic(models.Model):
```

Comment:
Shouldn't topics belong to a community instead of being global?

Reply:
Ok, but as things sit now, there will only ever be one set of topics and they are very specific to CZO.  I think we need to decide whether topics will be available to all communities or whether it's something that exists only for CZO.

Reply:
The ODM2 variable names are not specific to CZOs. They are specific concepts for the names of observed variables. The CZOs are using them, but others, but others might want to as well. However - they are just names of observed variables, and keywords may be selected from many other types of concepts - e.g., place names, project names, etc.

Reply:
Topics is a separate functionality from the odm2 variable names, as I understand it.

Reply:
> I think we need to decide whether topics will be available to all communities or whether it's something that exists only for CZO.

At least one more Community will use these Topics, as-is. The possibility exists that additional Communities may also use these Topics. As of now, all Communities use these Topics (one Community). When there are two Communities, it's likely "all Communities will be using those shared Topics". Depending on if the 3rd and 4th Communities use these Topics, at that time, the statement "all Communities will be using those shared Topics" may be True or False.

Reply:
Topics are not related to ODM2 or use of ODM2 in Subject Keyword. Topics are used in Resource Titles for Critical Zone Resources, for increased standardization and rapidly viewing/assessing long-lists of Titles.

Reply:
It has been decided that Topics will stay global for now, resolving